client/components/feature-modules/blocks/context/block-renderer-provider.tsx

```
@@ -18,69 +27,83 @@ export const RenderElementContext = createContext<{ widget: { id: string } } | n
+            // TODO: Support Entity Reference Listss
+            return <>Ill deal with this soon</>;
+        }
+
+        if (isContentNode(node) && isContentMetadata(node.block.payload)) {
+            const config = node.block.payload.listConfig;
+            if (!config) return null;
+            return (
+                <ContentBlockList
+                    id={node.block.id}
+                    config={config}
+                    children={node.children}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)